plugin/pkg/plugins.go

```diff
@@ -111,8 +111,8 @@ func AllPluginNames() []string {
 }
 
 // DefaultOnPlugins is the set of admission plugins that are enabled by default.
-func DefaultOnPlugins() sets.String {
-	return sets.NewString(
+func DefaultOnPlugins() sets.Set[string] {
+	return sets.New[string](
```

Comment:
```suggestion
	return sets.New(
```

Reply:
This doesn't seem to work 👀

Reply:
Can you check again? type param needs to be explicitly instantiated if given items are empty which is not the case here.